src/main/java/mekanism/common/tile/TileEntityTeleporter.java

```diff
@@ -58,6 +60,7 @@
     @Nullable
     private Direction frameDirection;
     private boolean frameRotated;
+    private EnumColor color;
```

Comment:
I am somewhat unsure of having the color also be stored in the tile as well as in the frequency. I am not quite sure on the best solution though so may look into it some myself when I am testing this PR and maybe try to move some of it into the frequency component (or maybe a subclass not sure yet).

Reply:
It is like this in the qio too.

Reply:
I will evaluate both at once then, given having it have a more proper home in the frequency component may make sense then.